openni2_camera/src/openni2_driver.cpp

```diff
@@ -398,7 +398,6 @@ void OpenNI2Driver::colorConnectCb()
   }
   std::lock_guard<std::mutex> lock(connect_mutex_);
 
-  // This does not appear to work
```

Comment:
these comments were left over and should have been removed when lazy subscribers were implemented